dev.txt

```diff
@@ -0,0 +1,7 @@
+coverage==4.4.1
+dj-database-url==0.4.2
+Django==1.11.6
```

Comment:
We need not repeat this in dev.txt. You can start with ```-r prod.txt``` in dev.txt. Have all but ```coverage``` in prod.txt. Just have ```coverage``` here. Also merge #171 and #172. I will close #172. Then we are good to go with this!